src/md.cc

```diff
@@ -674,5 +674,146 @@ void MGmol<OrbitalsType>::md(OrbitalsType** orbitals, Ions& ions)
     delete orbitals_extrapol_;
 }
 
+template <class OrbitalsType>
```

Comment:
I need to better understand what the purpose of this function is: if it is just to load functions from an HDF5 file, it does too many things, some that may be useless, others that may affect the runs unexpectedly (like "moveVnuc"). You can probably remove everything that is in if(ct.isLocMode()){}

Reply:
Currently this is only used for ROM snapshot collection, so no further simulation is executed after this routine. However, I agree that, as it is pushed to `release` branch as the "common" part, some others may use it somewhere else and receive some unexpected behavior.

I tested some parts of this routine to check if they are really necessary, and comment them if not.